torch/ao/pruning/_experimental/pruner/fpgm_sparsifier.py

```diff
@@ -0,0 +1,123 @@
+from typing import Callable, Optional, Union
+
+import torch
+
+from torch.ao.pruning._experimental.pruner import BaseStructuredSparsifier
+
+__all__ = ["FPGMSparsifier"]
+
+
+class FPGMSparsifier(BaseStructuredSparsifier):
+    r"""Filter Pruning via Geometric Median (FPGM) Structured Sparsifier
+    This sparsifier prune fliter (row) in a tensor according to distances among filters according to
+    "Filter Pruning via Geometric Median for Deep Convolutional Neural Networks Acceleration"
+    https://arxiv.org/abs/1811.00250.
+
+    This sparsifier is controlled by three variables:
+    1. `sparsity_level` defines the number of filters (rows) that are zeroed-out.
+    2. `dim` defines the dimension of the tensor to prune. Default: 0 (filter pruning).
```

Comment:
I think the paper only supports Filter pruning, which is also true of our mask parameterization implementation. Does the paper also mention kernel / subkernel pruning as well? If not let's remove this extra parameter and the code used to handle it